lms/templates/edraak_forus/style.css

```diff
@@ -0,0 +1,13 @@
+.form-field[class*=hidden] {
```

Comment:
CSS? I don't think it's a good idea. Just keep it inline then.

Or make it a first-class citizen SCSS file within the pipeline to ensure static file handling.